### DISCUSSION TITLE
Tarea 2 Parte 1 - Duda largo máximo de substring

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#142] [Tarea 2] [answered]
Debido a mi modelación requiero saber el valor del largo del substring máximo a buscar tal que mi código siga funcionando para este tamaño, debido a cálculos que realizo en mi solución.

**Comment by benjavicente** [comment, marked as answer]
Pueden asumir que el largo del substring es menor o igual a 31. El tamaño del string original y las consultas no puede ser asumido.

**Reply by MaxAl100** [reply]
Excelente! No necesito cambiar mi solución entonces :)
